src/Extension.hs

```diff
@@ -41,3 +47,30 @@ toHseEnabledExtensions :: [Extension] -> [HSE.Extension]
 toHseEnabledExtensions = mapMaybe ((HSE.EnableExtension <$>) . readMaybe . show')
   where
     show' e = if ex == "Cpp" then "CPP" else ex where ex = show e
+
+-- Oprhan instance. Bad.
+instance Ord Extension where
+  compare = compare `on` show
```

Comment:
Should this move into ghc-lib-parser-ex? Then it can be deleted at the appropriate time that it gets upstreamed to GHC?

It should also probably use the compare on fromEnum rather than show.

Reply:
Yes, let's move it down.

Reply:
Done in https://github.com/shayne-fletcher/ghc-lib-parser-ex/pull/31 and https://github.com/ndmitchell/hlint/pull/914